validation/views.py

```diff
@@ -59,7 +68,92 @@ def search_phrases(request):
     """
     The search results for pages.
     """
-    return HttpResponse(501)
+    query = request.GET.get("query")
+    cree_matches = Phrase.objects.filter(transcription__contains=query)
+    english_matches = Phrase.objects.filter(translation__contains=query)
+    all_matches = list(set().union(cree_matches, english_matches))
```

Comment:
You may want to use `sorted()` so that the order of `all_matches` is consistent between restarts of the Django server. The `set()` built-in is literally random, even Python 3.6 made dictionaries iterate in insertion order.